.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```

Comment:
Avoid opening two HTML files in the Live server at the same time. 
This causes a change in setting.json.